.github/workflows/test.yml

```diff
@@ -0,0 +1,58 @@
+name: Test
+
+on:
+  # Run each time we push and pull requests
+  push:
+  pull_request:
+  # Cron job
+  # https://help.github.com/en/actions/automating-your-workflow-with-github-actions/events-that-trigger-workflows#scheduled-events-schedule
+  schedule:
+    # https://crontab.guru/#0_0_1_*_*
+    - cron: "0 0 1 * *"
```

Comment:
Do we want a scheduled CI run? I think runs on push and pull-request are probably sufficient.

Reply:
I've occasionally encountered problems with incompatibilities due to package updates from either the distro or pip causing installation errors which can be hard to debug. As the project is only updated once in a while I think it would be nice to get a warning if anything breaks. On the other hand, if it causes a lot of unneeded compute (stealing from other `boutproject` quota I'm in favour of removing it as well 🙂.

Reply:
Good points. And the tests here shouldn't take long. :+1: for keeping.